lib/blocs/authentication/viewmodels/authentication_bloc.dart

```diff
@@ -22,19 +22,28 @@ class AuthenticationBloc extends Bloc<AuthenticationEvent, AuthenticationState>
     if (event is UnlockWallet) {
       yield state.copyWith(authStatus: AuthStatus.unlocked);
     }
+    if (event is OnCreateAccount) {
+      settingsStorage.saveAccount(event.account, event.privateKey);
+      // New account --> re-start auth status
+      add(const InitAuthStatus());
+      // Set fcm token must be last instruction to allow login, even if there is an error here.
+      await FirebaseMessageTokenRepository().setFirebaseMessageToken(settingsStorage.accountName);
+    }
     if (event is OnImportAccount) {
       settingsStorage.saveAccount(event.account, event.privateKey);
-      settingsStorage.privateKeyBackedUp = true;
+      settingsStorage.privateKeyBackedUp = false;
```

Comment:
on import private key backed up == true, so leave this as it was.

Reply:
private key backed up

Invite create account: false
recover account: False
import account using a private key: True - the user imported the account with a private key, meaning they have the private key stored somewhere outside the app already.